src/components/tipRecords/TipRecord.vue

```diff
@@ -20,9 +20,20 @@
         <a :href="tip.url" target="_blank" v-on:click.stop>
         <div class="tip__article--hasresults">
           <img v-bind:src="tip.preview.image" class="mr-2">
-          <span>
-            {{tipText}}
-          </span>
+          <div class="tip__article__content">
+            <h2 class="title text-ellipsis" 
+              :title="tipPreviewTitle">
+              {{tipPreviewTitle}}
+            </h2>
+            <div class="description"
```

Comment:
do we need text-ellipsis here too?

Reply:
The description is being cut off with ellipsis after specific number of rows